pkg/scaffold/controller/controllertest.go

```diff
@@ -113,7 +114,12 @@ func TestReconcile(t *testing.T) {
 	defer close(StartTestManager(mgr, g))
 
 	// Create the {{ .Resource.Kind }} object and expect the Reconcile{{ if .Resource.CreateExampleReconcileBody }} and Deployment to be created{{ end }}
-	g.Expect(c.Create(context.TODO(), instance)).To(gomega.Succeed())
+	err = c.Create(context.TODO(), instance)
+	// If getting an Invalid error, you should
+	// 1) modify the "instance" object to make it valid e.g. adding required fields.
+	// 2) remove the following if statement.
```

Comment:
May be state the reasoning first to get user's attention:
The `instance` object may not be a valid object because it might be missing some required fields. Modify the `instance` object to add required fields and then uncomment the following statement (have the line with error being nil assertion) ?

Reply:
Do you mean we should comment everything below?

IMO we should let it return early when encountering an Invalid error. IOW we should run everything if no Invalid error.

Reply:
To make it clear, IMO if we get an error from `Create`, it doesn't make a lot of sense to run the remaining test.